apps/notification/mutation.py

```diff
@@ -44,5 +44,37 @@ def mutate(root, info, data):
         return NotificationStatusUpdate(result=instance, ok=True, errors=None)
 
 
+class AssignmentStatusUpdate(PsGrapheneMutation):
+    class Arguments:
+        id = graphene.ID(required=True)
+        is_done = graphene.Boolean(required=True)
+    model = Assignment
+    result = graphene.Field(AssignmentType)
+    permissions = []
+
+    @classmethod
+    def perform_mutate(cls, root, info, **kwargs):
+        instance = cls.model.objects.get(id=kwargs['id'])
+        instance.is_done = kwargs['is_done']
+        instance.save(update_fields=['is_done'])
+        return cls(result=instance, ok=True)
+
+
+class AsssignmentBulkStatusUpdate(PsGrapheneMutation):
+    class Arguments:
+        is_done = graphene.Boolean(required=True)
+    model = Assignment
+    result = graphene.List(graphene.NonNull(AssignmentType))
+    permissions = []
+
+    @classmethod
+    def perform_mutate(cls, root, info, **kwargs):
+        instance = cls.model.objects.filter(created_for=info.context.user, is_done=False)
+        instance.update(is_done=kwargs['is_done'])
+        return cls(result=instance, ok=True)
```

Comment:
Looking at the codebase/UI we only need action for is_done=True, Let's do that only.

Also, we don't need to return anything, as this will update a lot of objects.

Also, PsGrapheneMutation is for entities related to project. Use GrapheneMutation here

AsssignmentMarkAllAsReadUpdate should be fine for this.